2017/01/09 08:27:08 [error] 25858#25858: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.101, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.50.102"
2017/01/09 08:27:09 [error] 25858#25858: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.101, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.50.102"
2017/01/09 08:30:18 [error] 25858#25858: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.101, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.50.102"
2017/01/09 08:30:45 [error] 25885#25885: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.101, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.50.102"
2017/01/09 08:30:48 [error] 25885#25885: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.50.101, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.50.102"
